docs/tutorials/tokens/erc1155.md

```diff
@@ -172,36 +172,14 @@ You can also approve other wallets to transfer tokens on your behalf with the `/
 - `operator`: The other account that is allowed to transfer tokens out of the wallet specified in the `key` field
```

Comment:
Need to update the default for "approved" above.

Reply:
Thanks. Good catch.